dist/ethereum.js

```diff
@@ -1037,7 +1043,36 @@ var isBoolean = function (object) {
  * @return {Boolean}
  */
 var isArray = function (object) {
-    return object instanceof Array; 
+    return object instanceof Array;
+};
+
+/** The following is a fix to be able to test as well using PhantomJS
+ * PhantomJS doesn't support bind yet: https://github.com/ariya/phantomjs/issues/10522
+ * As of PhantomJS 1.9.8, this is still required.
+ * @method phantomjsFix
+ */
+var phantomjsFix = function () {
```

Comment:
yep, I created #141 because it would be indeed nice to do so once the issue is fixed with the launcher.